src/store/categoriesSlice/slice.ts

```diff
@@ -0,0 +1,48 @@
+import { createSlice, createAsyncThunk } from "@reduxjs/toolkit";
+import { LOAD_STATUSES } from "../../constants";
+import { Category, getCategories } from "api/api";
+
+export interface State {
+  categories: { categories: Category[] };
+  loadStatus: LOAD_STATUSES;
+}
+
+const initialState: State = {
+  categories: {
+    categories: [
```

Comment:
просто пустой массив
```suggestion
    categories: [],
```